app/controllers/company/opening.js

```diff
@@ -41,7 +41,25 @@ function newOpening(req, res, next) {
     res.status(200).json(fetchedInstance);
   });
 }
-
+/**
+ * Function to update opening Details
+ * @param {Object} req Request Object
+ * @param {Object} res Response Object
+ * @param {Function} next Function to pass control to the next middleware
+ */
+function updateOpeningDetails(req, res, next) {
+  openingLib.updateJobOpening(req.user, req.body, function(
+    errInUpdation,
+    updatedInstance
+  ) {
+    if (errInUpdation) {
+      res.status(500).json(errInUpdation);
+    } else {
+      res.status(200).json(updatedInstance);
+    }
+  });
+}
 router.post('/addCredits', addCredits);
 router.post('/new', newOpening);
+router.post('/update', updateOpeningDetails);
```

Comment:
Ravi has given a suggestion on SH - Backend to put request type based on its usage
`POST` - new data ( '/', `newFunction` )
`PUT` - update data ( '/', `updateFunction` )
`DELETE` - delete data ( '/', `deleteFunction` )